setup.py

```diff
@@ -1,10 +1,11 @@
 from distutils.core import setup
+from setuptools import find_packages
 from temlogger import __version__
 
 
 setup(
     name='temlogger',
-    packages=['temlogger'],
+    packages=find_packages(exclude=[]),
```

Comment:
Do we need to pass `exclude=[]`? Ins't it the default value for `exclude`anyway?